docs/python/python-language.md

```diff
@@ -1240,6 +1305,46 @@ jobs = list(staff.values())
 print(jobs)  # ['Curriculum Developer', 'Curriculum Instructor', 'Designer']
 ```
 
+##### `open()`
```

Comment:
@alexgeer can we show the version that uses context managers here as well since we only use that style in the Python Language Track courses for File I/O?

```python
#opening "my_file.txt" in read mode
with open("my_file.txt", "rt") as file:
    contents = file.read()
    print(contents)

```

I get that `open()` and `close()` may be somewhat easier to digest for beginners, but it might be weird for the Python Language Track kids to come here and see it when they only use the `with` way.  I can write those docs in if you don't want to add it, just figured I'd let you take a run if you wanted.

Reply:
As a side note, we should ask @usrbinsam if there's any danger in having a bunch of open file descriptors around if kids use the `open()`/`close()` idiom and forget to run `close()` on the file handle. Not sure if that's something that would cause havoc on the backend, but would be good to run by him.

Reply:
I'm not sure what to do it. The context-based way is standard but not basic so I don't want to do the cart-before-horse thing that we have been so guilty of in curriculum before. I'd want them to know the basic operations going on behind the context management before they use it, even though it's easier. I will probably add the context management to the early lesson content, but imo we should have both in the docs.